rpi_backlight/cli.py

```diff
@@ -48,19 +53,37 @@ def _create_argument_parser():
         action="version",
         version="%(prog)s {version}".format(version=__version__),
     )
+    parser.add_argument(
+        '-B', '--board', default=1,
+        help='Please provide a board model.\n'
+        '0: Raspberry Pi\n'
+        '1: TinkerBoard\n.')
     return parser
 
-
 def main():
     """Start the command line interface."""
     parser = _create_argument_parser()
     args = parser.parse_args()
 
-    if args.sysfs_path is not None:
+    if args.board:
+        board = int(args.board)
+        if (board != 1 and board != 2):
+            parser.error("Board parameter must be either 1 (Raspberry Pi) or 2 (TinkerBoard)")
+
+        if (board == 2 and args.sysfs_path is not None):
+            backlight = Backlight(board_type=BoardType.TINKER_BOARD,backlight_sysfs_path=args.sysfs_path)
+        elif (board == 2 and args.sysfs_path is None):
+            backlight = Backlight(board_type=BoardType.TINKER_BOARD,backlight_sysfs_path="/sys/devices/platform/ff150000.i2c/i2c-3/3-0045/")
+        elif (board == 1 and args.sysfs_path is not None):
+            backlight = Backlight(backlight_sysfs_path=args.sysfs_path)
+        elif (board == 1 and args.sysfs_path is None):
+            backlight = Backlight()
+    elif (args.sysfs_path is not None):
         backlight = Backlight(backlight_sysfs_path=args.sysfs_path)
-    else:
+    else: 
         backlight = Backlight()
```

Comment:
Determining the value for `backlight_sysfs_path` is now handled by the constructor, so this can be simplified to only set `board_type` based on `-B`/`--board`.

Reply:
Fixed (I think)